mlx/primitives.cpp

```diff
@@ -977,7 +977,7 @@ std::vector<array> Convolution::vjp(
           }
         }
 
-        grad = slice(grad, std::move(starts), std::move(stops), stream());
+        grad = slice(grad, starts, stops, stream());
```

Comment:
Slice takes its arguments by value. What we should actually do is check if there are places that could pass by move yet pass by copy.